### DISCUSSION TITLE
New apps suggestions

### DISCUSSION THREAD

**Discussion by Psyhackological** [Psyhackological/AAA#43] [Ideas]
# Hello there

This is a section where you can list your app that in your opinion should be on the list. Remember to follow [the rules](https://github.com/Psyhackological/AAA#rules=) and [Contribution Guidelines](https://github.com/Psyhackological/AAA/blob/main/CONTRIBUTING.md).

**Comment by rahulduhan** [comment]
~~[Kiwix](https://www.kiwix.org/en/)~~ 
Added

**Comment by rahulduhan** [comment]
[Flym](https://github.com/FredJul/Flym) 
similar to feeder
not updated anymore but modern looking with dark mode

**Reply by Psyhackological** [reply]
If it is a smooth experience without crashes or error you can add it.

**Reply by rahulduhan** [reply]
yes, but read [`this`](https://github.com/FredJul/Flym#beware)

**Reply by Psyhackological** [reply]
So only F-Droid?

**Reply by rahulduhan** [reply]
Updates are blocked on play store 
so the dev stopped the development

**Comment by rahulduhan** [comment]
need help with ~~blade player~~ (Added)
It could function without any account using local source for music 
but it also requires a account to stream music from spotify
as I don't have spotify account , could anyone test this app ? (it is available on [playstore](https://play.google.com/store/apps/details?id=v.blade))

**Comment by rahulduhan** [comment]
snapdrop -> I have hear that this app works like airdrope on the susphone but I couldn't make it work on my PC. May be it is librewolf my browser or my phone casuing some bug 
So could anyone test this app ?

**Reply by Psyhackological** [reply]
Already added.

**Comment by rahulduhan** [comment]


**Reply by rahulduhan** [reply]
There are two trackers (crash analytics and firebase) in this app but still the app is licenced under GPL

**Reply by Psyhackological** [reply]
Let's wait with this one. I need to work on this Privacy thing. See here: #17.

**Reply by rahulduhan** [reply]
OK
my bad didn't see till the end of that issue

**Reply by Psyhackological** [reply]
That is okay. Still idk what to do with it.

**Reply by rahulduhan** [reply]
The app is very good-looking
I might remove the proprietary bits in future when I learn some app dev

**Comment by rahulduhan** [comment]
~~[simple login](https://simplelogin.io/)~~
Added

**Comment by rahulduhan** [comment]
~~[joplin](https://joplinapp.org/)~~
Added

**Comment by rahulduhan** [comment]
~~[KAKUKO](https://github.com/blastrock/kakugo)~~
Edit :  Added

**Reply by Psyhackological** [reply]
Language maybe
Or Knowledge

**Comment by SleepDaemon** [comment]
Delta Icon Pack (Links: [Official Site](https://delta-icons.github.io/), [Github](https://github.com/Delta-Icons/android), [Google Play](https://play.google.com/store/apps/details?id=website.leifs.delta), [IzzyDroid](https://apt.izzysoft.de/fdroid/index/apk/website.leifs.delta))

Harmonoid Music Player/YTM Streaming (Links: [Github](https://github.com/harmonoid/harmonoid)) P.S the android version is having a rework at the time of writing this, but should be out maybe in 2-3 months.

Trail Sense (Links: [Official Site](https://kylecorry.com/Trail-Sense/), [Github](https://github.com/kylecorry31/Trail-Sense), [F-Droid](https://f-droid.org/en/packages/com.kylecorry.trail_sense/), [Google Play](https://play.google.com/store/apps/details?id=com.kylecorry.trail_sense))

FakeStandby (Links: [Github](https://github.com/JonasBernard/FakeStandby), [F-Droid](https://f-droid.org/packages/android.jonas.fakestandby/))

SystemUI Tuner (Links: [Official Site](https://zwander.dev/dialog-systemuituner), [Google Play](https://play.google.com/store/apps/details?id=com.zacharee1.systemuituner), [Github](https://github.com/zacharee/SystemUITunerRedesign))

CamWings: Anti-Camera (Links: [Official Site](https://schiffer.tech/camwings-mobile.html), [Google Play](https://play.google.com/store/apps/details?id=com.schiffertech.camwings)) {Android 12 has this built in but this is for those who don't have android 12}

Animity (Links: [Github](https://github.com/kl3jvi/animity))

**Reply by Psyhackological** [reply]
To check or add (I'm going to update this with time): 
- [x] Delta Icon Pack (Links: [Official Site](https://delta-icons.github.io/), [Github](https://github.com/Delta-Icons/android), [Google Play](https://play.google.com/store/apps/details?id=website.leifs.delta), [IzzyDroid](https://apt.izzysoft.de/fdroid/index/apk/website.leifs.delta))
- [ ] Harmonoid Music Player/YTM Streaming (Links: [Github](https://github.com/harmonoid/harmonoid)) P.S the android version is having a rework at the time of writing this, but should be out maybe in 2-3 months.
- [x] Trail Sense (Links: [Official Site](https://kylecorry.com/Trail-Sense/), [Github](https://github.com/kylecorry31/Trail-Sense), [F-Droid](https://f-droid.org/en/packages/com.kylecorry.trail_sense/), [Google Play](https://play.google.com/store/apps/details?id=com.kylecorry.trail_sense))
- [ ] FakeStandby (Links: [Github](https://github.com/JonasBernard/FakeStandby), [F-Droid](https://f-droid.org/packages/android.jonas.fakestandby/))
- [ ] SystemUI Tuner (Links: [Official Site](https://zwander.dev/dialog-systemuituner), [Google Play](https://play.google.com/store/apps/details?id=com.zacharee1.systemuituner), [Github](https://github.com/zacharee/SystemUITunerRedesign))
- [ ] CamWings: Anti-Camera (Links: [Official Site](https://schiffer.tech/camwings-mobile.html), [Google Play](https://play.google.com/store/apps/details?id=com.schiffertech.camwings)) {Android 12 has this built in but this is for those who don't have android 12}
- [ ] Animity (Links: [Github](https://github.com/kl3jvi/animity))

**Comment by Psyhackological** [comment]
~~[UniCiv](https://github.com/yairm210/Unciv)~~

Edit: added

**Reply by rahulduhan** [reply]
Game?

**Reply by Psyhackological** [reply]
As it seems.

**Comment by Psyhackological** [comment]
~~[DeepLAndroid](https://github.com/sakusaku3939/DeepLAndroid)~~

Edit: added.

**Comment by rahulduhan** [comment]


**Reply by Psyhackological** [reply]
Already added.

**Comment by rahulduhan** [comment]
Which category should fakestandby go ?
Android customizations ?

**Reply by Psyhackological** [reply]
I think so, but it seems like a "Android Hack". 😆

**Comment by MrCamel999** [comment]
QUIK
A continuation of QKSMS since QKSMS is no longer developed. On that note, I also feel QKSMS should be removed from the list.

**Reply by Psyhackological** [reply]
Sure, maybe the issue should be the more correct place to report it, but I agree.

**Reply by Psyhackological** [reply]
@MrCamel999 [this](https://github.com/Psyhackological/AAA/commit/8f03609fd49a92d18ca29e7875bf82e0ff2b5c79) should do the job. :)

**Comment by Psyhackological** [comment]
Closing this old discussion. I created a project with [Suggestion of apps](https://github.com/users/Psyhackological/projects/1/views/1?query=is%3Aopen+sort%3Aupdated-desc). If anyone wants to suggest something, just tell me through discussions, issues or on Lemmy or other way. Preferably with Pull Request.
